packages/logs/src/boot/startLogs.ts

```diff
@@ -75,6 +82,7 @@ export function startLogs(configuration: LogsConfiguration, getCommonContext: ()
     startLogsBridge(lifeCycle)
   }
 
+  addTelemetryConfiguration(serializeLogsConfiguration(initConfiguration))
```

Comment:
**🥜 nitpick:** ‏what about colocating this code with other code related to telemetry?